### DISCUSSION TITLE
Cost Override Obsolete - Help with Resetting to Current Tariff

### DISCUSSION THREAD

**Discussion by mth211** [BottlecapDave/HomeAssistant-OctopusEnergy#907] [Q&A] [answered]
Updating to 11.1.0 resulted in a repair required for deprecated cost override sensors. Following the guide to the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/repairs/cost_override_obsolete/) gives me a good hint, but can't for the life of me actually work out what I need to do to repair the issue. Any pointers on what entity I actually need to edit, and to what?

**Comment by BottlecapDave** [comment, marked as answer]
Hello. You'll have a sensor called tariff override for each active meter. This is what you set to the tariff that you wanted to track. You need to set this to the tariff code that can be found in your corresponding current rate sensor to remove the repair warning. You then need to follow the link in the docs to setup the new tariff comparison sensors.

**Comment by mth211** [comment]
Thanks @BottlecapDave. the bit I'd not twigged was needing to get the tarrif code from the sensor. I'd forgotten it was one of the attributes and every one of the versions of the code I tried from the API wasn't doing anything, and the integration was quite rightly not accepting anything not in the correct format

All resolved now, thanks
